src/compiler/scala/tools/nsc/javac/JavaParsers.scala

```diff
@@ -752,11 +752,6 @@ trait JavaParsers extends ast.parser.ParsersCommon with JavaScanners {
                   makeTemplate(List(), statics))
       }
 
-    def importCompanionObject(cdef: ClassDef): Tree =
-      atPos(cdef.pos) {
-        Import(Ident(cdef.name.toTermName), ImportSelector.wildList)
-      }
-
```

Comment:
Is this intentional?

Reply:
yeah, it's unused since https://github.com/scala/scala/commit/4e822d74be